activerecord/test/cases/arel/attributes/attribute_test.rb

```diff
@@ -639,6 +639,16 @@ class AttributeTest < Arel::Spec
           )
         end
 
+        it "can be constructed with a range implicitly ending at Infinity" do
+          attribute = Attribute.new nil, nil
+          node = attribute.between(0..)
```

Comment:
I think this will give us a syntax error on 2.5.  Can you conditionalize this test so we don't get build failures on 2.5?

Reply:
Sure. The built actually failed for precisely this reason. I'll push an update tomorrow.

Reply:
@tenderlove just pushed an update.

Reply:
D'oh, what I did won't work for syntax error :see_no_evil:

I can either:

1. Come up with something that would work for syntax errors.
2. Leave the check as it is and replace `0..` with `0..nil` (they're equivalent). In Rubyies pre-2.6 the latter results in an `ArgumentError` exception.

@tenderlove what do you think?